php/libraries/FeedbackMRI.class.inc

```diff
@@ -57,6 +57,27 @@ class FeedbackMRI
         }
     }
 
+    /**
+     * Clears all predefined comments from the object
+     * (used prior to re-setting the list of predefined comments)
+     *
+     * @return bool true if operation succeeded
+     */
+    function clearPredefinedComments()
+    {
+        // start the query
+        $query = "DELETE FROM feedback_mri_comments WHERE ".$this->_buildWhere();
+
+        // limit to predefined comments
+        $query .= "AND PredefinedCommentID IS NOT NULL";
+
+        // run the query
+        $success = mysql_query($query);
```

Comment:
Didn't you remove this in a previous PR since mysql_query hasn't existed since PHP 7? Why is it getting added here?